eugenepunov/lab1-BasicIO/user.h

```diff
@@ -0,0 +1,28 @@
+/******************************************************************************/
+/* User Level #define Macros                                                  */
+/******************************************************************************/
+
+// Add I/O name macros here
+#define led1    LATGbits.LATG6
```

Comment:
Why not use `LED1` for defined stuff here? It's common practice (see the `LATG6`)